sig-network/charter.md

```diff
@@ -30,7 +30,7 @@ The following topics fall under ownership of this SIG:
   - APIs for defining L3/4 loadbalancing (e.g. [Service])
   - Reference implementations (e.g. [kube-proxy]).
 - Ingress
-  - APIs for defining L7 loadbalancing.
+  - APIs for defining L7 loadbalancing (e.g. [Ingress], [Gateway API])
```

Comment:
Gateway (and Ingress?) isn't just L7.

I think of them as "Cluster ingress APIs", but maybe that's also too narrow for Gateway these days?

Maybe they just belong under the "Services" heading? (If you're going to add Gateway, you should probably at least rename the "Ingress" heading anyway?)

Reply:
I've tried a slightly different wording and organization, lmkwyt